content/api/guides/appownerintegration/receivingdata/_index.md

```diff
@@ -167,7 +166,7 @@ Ved å benytte sitt tjenesteeier token vil systemet kunne laste ned instance doc
 
 I instance dokumentet fra steget over er det listet dataelementene som en instance består av. 
 
-Disse dokumentene kan lastes ned fra applikasjons endepunkt. Hver dataelement har informasjon om f.eks datatype og når det var sist endret.
+Disse dokumentene kan lastes ned fra applikasjons endepunkt. Hvert dataelement har informasjon om f.eks datatype og når det var sist endret.
```

Comment:
Vil det gi mening å gjenta json-eksempelet kun for et dataelement her og evt. highlighte linjen som peker på url for å gjøre det enda tydeligere?